atomic-design.md

```diff
@@ -0,0 +1,101 @@
+
+## What's an atomic element?
+
+An atomic element is any self contained code pattern that follows the Atomic Design principles. The types of elements are atoms, molecules, organisms, and templates.
+
+### Atoms
+
+- The base unit
+- Think buttons, input fields and other html tags
+- Abstract and not too useful on their own
+
+### Molecules
+
+- It can do something
+- A series of atoms combined to serve a purpose
+
+### Organisms
+
+- Self contained chunks with different functionalities to form a distinct section
+- Think headers, comments section, etc
+
+### Templates
+
+- To us: literally which atoms, molecules and organisms live on the wagtail templates
+
+### How do I know what to make into an element?
+
+- If it can't stand on it's own. For example a paragraph or list item have minimal styles added to them and can live on their own so they wouldn't be elements. A button is more complex (comparatively) and so it should become an element.
+- If it's not a utility. Utilities are adjustments and helpers, they don't need to become elements.
+- If it's not a layout helper. Layout helpers are specifically used to lay out the structure of a page and are never concerned with the content within them.
+
+## File organization
+
+Both prefixing and organizing elements by their type is the easiest way to keep them organized. All element classes should be prefixed by the first letter of it's type (ex `.m-pagination`) and placed in a corresponding directory (ex. `/src/molecules/pagination.less`). To save yourself the trouble of moving things multiple times, we have found it easiest to break up a component in it's top level stylesheet, then move each class block to it's appropriate sub-directory.
+
+## Working with code
+
+### Install the Capital Framework Sandbox
+
+1. Follow the [installation instructions](https://github.com/jimmynotjim/capital-framework-sandbox#installation).
+2. Click around to ensure the local server is rendering the sandbox correctly.
+3. Stop the server `ctrl+c`.
+
+### Link your component
+
+1. From the root of `capital-framework`
+
+  `npm run cf-link`
+
+  _To add your component to this command, edit [the script] in `package.json`_
+2. In a new tab navigate to the root of the Sandbox
+
+   `cd capital-framework-sandbox`
+3. Run the same command from the root of the Sandbox repo
+
+  `npm run cf-link`
+
+  _Again, to add your component to this command, edit the script in this repos `package.json`_
+4. Run the build command
+
+   `npm run build`
+5. Start the server and check out the changes
+
+   `npm start`
+
+_When you’re done working on or reviewing a component, run `npm unlink component-name && npm install component-name`_
+
+### Converting a non-atomic component for the v4 release
+
+1. Checkout the [v4 prerelease branch](https://github.com/cfpb/capital-framework/tree/prerelease-ver-4) of Capital Framework
+
+  ```sh
+  cd /capital-framework
```

Comment:
If we're going to suggest a `cd` command here (and below for getting into `capital-framework-sandbox`), probably shouldn't lead with a slash. I doubt many folks keep their repos at the root of their filesystem ;)